docs/deployment/minikube.md

```diff
@@ -58,7 +58,9 @@ Kong is accesssible. This will be used to actually send reqeusts into the
 Kubernetes cluster.
 
 ```bash
-$ export PROXY_IP=$(minikube service -n kong kong-proxy --url | head -1)
+$ export PROXY_IP=$(minikube service kong-kong-proxy --url | head -1)
+# If installed by helm without specifying namespace, use the line below instead.
+# $ export PROXY_IP=$(minikube service -n kong kong-kong-proxy --url | head -1)
```

Comment:
If namespace is not specified in Helm installation, then why do we need to pass `-n kong`?

Reply:
Ah, I actually intended not to specifying the namespace, that's one reason of this PR.
I pushed the fix!